benchmark/fs/bench-readdir.js

```diff
@@ -5,16 +5,19 @@ const fs = require('fs');
 const path = require('path');
 
 const bench = common.createBenchmark(main, {
-  n: [1e4],
+  n: [10],
+  dir: [ 'lib', 'test/parallel'],
+  withFileTypes: ['true', 'false']
 });
 
-
-function main({ n }) {
+function main({ n, dir, withFileTypes }) {
+  withFileTypes = withFileTypes === 'true';
   bench.start();
   (function r(cntr) {
     if (cntr-- <= 0)
       return bench.end(n);
-    fs.readdir(path.resolve(__dirname, '../../lib/'), function() {
+    const fullPath = path.resolve(__dirname, '../../', dir);
```

Comment:
Maybe something for a separate commit or PR but you could move this to before the `bench.start()` call. The fewer tangents the benchmark has, the better.

(Ditto in the other file.)

Reply:
Done, thanks for spotting this! (Pretty sure I made a change for that before, somehow it got lost before I opened the PR..)